### PR TITLE
feat: Add Dropdown for Installation Commands on Homepage

### PR DESCRIPTION
**Pull request by Jacksonmills**
Closes [#1609](https://github.com/t3-oss/create-t3-app/issues/1609)

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Add dropdown for different package manager copy to clipboard options on homepage

---

## Screenshots

image
image

🫡💖

### CODE REVIEW

**Review by juliusmarminge — CHANGES_REQUESTED** [review]
This is great, but could you make it match the language selector more?

https://github.com/t3-oss/create-t3-app/assets/51714798/7dba469c-d282-45f2-a144-6f1c9581efa9

**Comment by Jacksonmills** [comment]
> This is great, but could you make it match the language selector more?
> CleanShot.2023-10-20.at.09.13.51.mp4

Ah yes! That would make a lot more sense, I will update that tomorrow 🫡

**Comment by ronanru** [comment]
I think bun command should be `bun create t3-app@latest`. Also, you don't need a changeset for docs site changes.

**Comment by Jacksonmills** [comment]
> I think bun command should be `bun create t3-app@latest`. Also, you don't need a changeset for docs site changes.

Okay noted, thanks! Bux command I got from the readme doc but maybe they both need updated?

I'll remove the change set once I push the headlessui change, making it much more inline with the docs style

**Comment by Jacksonmills** [comment]
image
image

Updated screenshots 🫡

**Comment by Jacksonmills** [comment]
> I think bun command should be `bun create t3-app@latest`. Also, you don't need a changeset for docs site changes.

I removed the changeset but now the CI is mad, is that something y'all just override once you review?

**Comment by Jacksonmills** [comment]
updated styles to remove weird rounded edges of each item

image
image

**Comment by Jacksonmills** [comment]
Something I am noticing is that `bg-default` seems to have quite different colors for the docs homepage vs the landing page so I am unable to perfect the color to match the dark theme docs dropdown.

image
image

**Comment by Jacksonmills** [comment]
Fixed!
image
image

**Review by juliusmarminge — APPROVED** [review]
Lgtm

**Review by juliusmarminge — COMMENTED** [review]
bun create t3-app
bun

?

File: www/src/components/landingPage/ClipboardSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  {
+    command: "yarn create t3-app",
+    manager: "yarn",
+  },
+  {
+    command: "pnpm create t3-app@latest",
+    manager: "pnpm",
+  },
+  {
+    command: "bunx create-t3-app@latest",
```
Comment (juliusmarminge) [inline www/src/components/landingPage/ClipboardSelect.tsx:19]: bun?

Comment (Jacksonmills) [reply www/src/components/landingPage/ClipboardSelect.tsx:19]: in the README.md it says `bunx`

Comment (Jacksonmills) [reply www/src/components/landingPage/ClipboardSelect.tsx:19]: https://github.com/t3-oss/create-t3-app#bun

Comment (juliusmarminge) [reply www/src/components/landingPage/ClipboardSelect.tsx:19]: That should be updated too

Comment (Jacksonmills) [reply www/src/components/landingPage/ClipboardSelect.tsx:19]: okay I can add that to this PR then? Updating now!

Comment (Jacksonmills) [reply www/src/components/landingPage/ClipboardSelect.tsx]: Added!

**Comment by juliusmarminge** [comment]
Sorry for the confusion, but we still want the @latest - my bad for being unclear 😐

**Comment by Jacksonmills** [comment]
> Sorry for the confusion, but we still want the @latest - my bad for being unclear 😐

xD np I'll update in a bit.

**Comment by joachimjusth** [comment]
Lgtm

**Review by juliusmarminge — APPROVED** [review]
Thank you for your patience

**Comment by Jacksonmills** [comment]
> Thank you for your patience

np! stoked you guys liked the change, hope to contribute more soon 🫡💖

**Pull request merged**